### ISSUE TITLE
Do not aggregate FieldDescriptors for different content types

### ISSUE THREAD

**Issue by mduesterhoeft** [ePages-de/restdocs-api-spec#26]
When aggregating FieldDescriptors for different requests with the same path and method we now always aggregate the FieldDescriptors from all requests and create a JsonSchema. In case we have requests/response for different content types we should not do this.

A good example is `PATCH` `/products/{id}` in the sample app - here we have two requests with `application/json-patch+json` and `application/json` and we currently generate this schema:

```yaml
type: "object"
    required:
    - "name"
    - "price"
    properties:
      '[]':
        type: "object"
        properties:
          path:
            type: "string"
            description: "The path of the field."
          op:
            type: "string"
            description: "Patch operation."
          value:
            type: "string"
            description: "The value to assign."
      price:
        type: "number"
        description: "The price of the product."
      name:
        type: "string"
        description: "The name of the product."
        minLength: 1
```

This is obviously wrong.

**Comment by lzhoucs** [comment]
This seems already supported as of `0.9.0`. In fact this is the feature that made us switch to this tool from springfox which doesn't support open api spec 3.

**Comment by CalamarBicefalo** [comment]
For anyone struggling @lzhoucs is right. Just bear in mind the identifiers of the different documents need to be different.

Annoyingly the spec will pic the last identifier as the operation identifier which makes no sense. But that is a different issue.

**Comment by ozscheyge** [comment]
> Annoyingly the spec will pic the last identifier as the operation identifier which makes no sense. But that is a different issue.

This should be addressed with https://github.com/ePages-de/restdocs-api-spec/issues/140 https://github.com/ePages-de/restdocs-api-spec/pull/141 , so this should be fixed as of version `0.10.0`. 
Combining this with the above comments:
* https://github.com/ePages-de/restdocs-api-spec/issues/26#issuecomment-487342635
* https://github.com/ePages-de/restdocs-api-spec/issues/26#issuecomment-677682826
...this whole issue seems to be fixed, or can be worked around by choosing distinct operationIds (snippet names).

Feel free to re-open, if this isn't true.
